src/components/Calculator.js

```diff
@@ -0,0 +1,19 @@
+import './stylesheet/calculator.css';
```

Comment:
- [ ] 👏 Great work on adding the `CSS` for your calculator! However, it appears that the styles you've implemented haven't loaded correctly in your `calculator.js` component. This issue is likely because of an incorrect import path for your `CSS` file. As a result, you may be encountering an error in the console indicating that the imported module can't be found. Below is the details of the error:
<img width="1098" alt="Screenshot 2023-09-16 at 00 55 47" src="https://github.com/Izebiafe/math-magician/assets/107192076/402d4fbf-21b3-4a87-8a13-33b2cc941d8c">


To resolve this issue, please make sure to import your `CSS` file correctly. Instead of: 👇👇

```javascript
import './stylesheet/calculator.css'
```

Please use the following import statement: 👇👇

```javascript
import '../stylesheet/calculator.css'
```

This corrected import path should ensure that your styles are properly loaded, allowing your calculator component to display as intended. Happy coding! 😊🚀🍀